CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 - **[Breaking]** `ValueType` and `GetType` functionality is removed in favor of using
   `reflect.Kind`
+- Update documentation: cleanup markup, trim readme and doc.go in
+favor of godoc.
```

Comment:
bad indent, should be two spaces here and also i don't think this is changelog worthy